tests/test_CentralityClasses.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+import numpy as np
+from sparkx.CentralityClasses import CentralityClasses
+
+
```

Comment:
What happens if I add centrality bins with a value greater 100? Should this be tested?

Reply:
I'll add more tests in the constructor.